topbin/dune

```diff
@@ -18,7 +24,10 @@
  (public_name coqc)
  (package coq-core)
  (modules coqc_bin)
- (libraries coq-core.toplevel))
+ ; we include the hash of coqworker.byte but actually depend on coqworker.opt
+ (link_deps %{project_root}/_build/install/%{context_name}/lib/coq-core/coqworker.opt%{ext_exe})
```

Comment:
It seems this dependency breaks dune when the coq rules for stdlib are present. Is there some workaround?

@ejgallego @rgrinberg 

(context: `coqc` calls `coqworker` at runtime in this PR)